auto_invitation_to_notion/main.py

```diff
@@ -0,0 +1,127 @@
+import requests
+import copy
+from collections import defaultdict
+from typing import List, Dict
+
+from notion.client import NotionClient
+
+
+LUBYCON_PRIVATE_PAGE = '4594ead1-b975-451d-a578-2ffeea6aa452'
+LUBYCON_MENTOR_PAGE = '720e43a5-f572-440a-8e58-847b72359b16'
+LUBYCON_HUB_PAGE = 'd3ebe34b-34af-4b2f-b984-ae1e91cff7f3'
+
+
+def get_lubycon_admin_uid(client: NotionClient) -> str:
+    return client.get_email_uid().get('lubycon@gmail.com')
+
+
+def get_workspace_id(client: NotionClient, admin_uid: str) -> str:
+    response = client.post('getSpaces', {})
+    if response.ok:
+        workspace_id = list(response.json()[admin_uid]['space'].keys())[-1]
+        return workspace_id
+    else:
+        raise Exception(handle_request_error(response=response))
+
+
+def handle_request_error(response: requests.Response) -> None:
+    raise Exception(
+        f"request fail!. status code: {response.status_code}, detail: {response.text}")
+
+
+def get_uid_pageids_assigned_users(client: NotionClient, workspace_id: str) -> Dict:
+    response = client.post("getSubscriptionData", {
+                           "spaceId": workspace_id, "version": "v2"})
+    if response.ok:
+        users_info = response.json()["users"]
+
+        answer = {}
+        for user_info in users_info:
+            answer[user_info["userId"]] = {
+                "guest_page_ids": [] if user_info.get("guestPageIds") is None else user_info.get("guestPageIds")}
+
+        return answer
+    else:
+        raise Exception(handle_request_error(response=response))
+
+
+def detect_authority(user_info: Dict) -> Dict:
+    d = copy.deepcopy(user_info)
```

Comment:
```suggestion
    deep_copied_user_info = copy.deepcopy(user_info)
```
요런 부분은 조금 길어지더라도 명확한 변수명을 사용하면 더 코드를 이해하기 쉬울 것 같습니다...!

Reply:
작은 함수 스코프에서는 크게 문제될 것이라고 생각되진 않아서 크게 상관없다고 생각은 하지만, 
제가 다른 변수의 이름은 또 가독성을 염두해두어서; 변경하도록 하겠습니다.

Reply:
반영 완료했습니다.